test/controllers/helpers/CheckYourAnswersHelperSpec.scala

```diff
@@ -32,6 +32,11 @@ class CheckYourAnswersHelperSpec extends SpecBase with SummaryListFluency {
   when(fakeMessages("site.change")) thenReturn "Change"
```

Comment:
Seems unnecessary to mock all the messages here? We can just use the stubbed messages in the SpecBase and check the result match the values like "site.change"

Reply:
i agree! -changed! :)

Reply:
actually - having checked your way makes the tests fail - if you can fix it better in your ticket feel free - but I'll pop this one in for now :)